geoapps/inversion/driver.py

```diff
@@ -398,96 +405,53 @@ def get_path(self, file):
-
-    elif inversion_type == "direct current":
-        from geoapps.inversion.electricals.direct_current.three_dimensions.constants import (
-            validations,
-        )
-        from geoapps.inversion.electricals.direct_current.three_dimensions.params import (
-            DirectCurrent3DParams as ParamClass,
-        )
-    elif inversion_type == "direct current 2d":
-        from geoapps.inversion.electricals.direct_current.two_dimensions.constants import (
-            validations,
-        )
-        from geoapps.inversion.electricals.direct_current.two_dimensions.params import (
-            DirectCurrent2DParams as ParamClass,
-        )
-
-    elif inversion_type == "induced polarization":
-        from geoapps.inversion.electricals import (
-            InducedPolarization3DParams as ParamClass,
-        )
-        from geoapps.inversion.electricals.induced_polarization.three_dimensions import (
-            InducedPolarization3DParams as ParamClass,
-        )
-        from geoapps.inversion.electricals.induced_polarization.three_dimensions.constants import (
-            validations,
-        )
-
-    elif inversion_type == "induced polarization 2d":
-        from geoapps.inversion.electricals.induced_polarization.two_dimensions import (
-            InducedPolarization2DParams as ParamClass,
-        )
-        from geoapps.inversion.electricals.induced_polarization.two_dimensions.constants import (
-            validations,
-        )
-
-    else:
-        raise UserWarning("A supported 'inversion_type' must be provided.")
-
-    input_file = InputFile.read_ui_json(filepath, validations=validations)
-    params = ParamClass(input_file=input_file, **kwargs)
-
-    driver = InversionDriver(params)
-
-    with params.geoh5.open(mode="r+"):
-        driver.run()
-
-    return driver
-
-
 if __name__ == "__main__":
-    start_inversion(sys.argv[1])
+
+    from geoapps.inversion.electricals.direct_current.three_dimensions.driver import (
+        DirectCurrent3DDriver,
+    )
+    from geoapps.inversion.electricals.direct_current.two_dimensions.driver import (
+        DirectCurrent2DDriver,
+    )
+    from geoapps.inversion.electricals.induced_polarization.three_dimensions.driver import (
+        InducedPolarization3DDriver,
+    )
+    from geoapps.inversion.electricals.induced_polarization.two_dimensions.driver import (
+        InducedPolarization2DDriver,
+    )
+    from geoapps.inversion.natural_sources.magnetotellurics.driver import (
+        MagnetotelluricsDriver,
+    )
+    from geoapps.inversion.natural_sources.tipper.driver import TipperDriver
+    from geoapps.inversion.potential_fields.gravity.driver import GravityDriver
+    from geoapps.inversion.potential_fields.magnetic_scalar.driver import (
+        MagneticScalarDriver,
+    )
+    from geoapps.inversion.potential_fields.magnetic_vector.driver import (
+        MagneticVectorDriver,
+    )
+
+    DRIVER_MAP = {
```

Comment:
We can probably stash all of this in the init of inversion, and just import DRIVER _MAP?

Reply:
Yep, seems to work if I import in the name=__main__.  Good idea.